pytorch_blade/.bazelrc

```diff
@@ -2,8 +2,7 @@ try-import ../tf_community/.bazelrc
 try-import ../tf_community/.bazelrc.user
 
 build --disk_cache=~/.cache
-build:torch_cuda --define enable_cuda=true --config=cuda
-build:torch_disc_cpu --config=disc_cpu --config=disc_mkldnn
-build:torch_disc_cuda --config=disc_cuda --config=torch_cuda
+build:torch_cuda --config=cuda --config=disc_cuda --define enable_cuda=true
+build:torch_cpu --config=disc_cpu --config=disc_mkldnn
```

Comment:
A little comment, shall we unify configuration names through out the whole project? If so, they're better to be `cuda` and `cpu` ? @wyzero any comments?

Reply:
PyTorch convention is to use cuda rather than gpu.

Reply:
yeah, I think a more specific name is better. In the future, we will support other types of GPUs and CPUs.

```
disc_x86
disc_aarch64
disc_cuda
disc_ppu
....
```

Reply:
So, I mean shall we just remove prefixes like `torch_` / `disc_`, use arch name for all modules.

Reply:
I think the renaming of the configuration option may be created in another pull request?